fobi_custom/plugins/pldp/form_elements/fields/study/forms.py

```diff
@@ -0,0 +1,48 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from fobi.base import BaseFormFieldPluginForm, get_theme
+from pldp.models import Study
+
+
+theme = get_theme(request=None, as_instance=True)
+
+class PLDPStudyForm(forms.Form, BaseFormFieldPluginForm):
```

Comment:
A Google doc for Gehl changes sounds great! Shall I start one? Multiple surveys can belong to a single study—why do you see the need for another tier?

Reply:
Yes, please start a doc! 

All right – [according to the PLDP study table, a study can be a part of a "project"](https://github.com/datamade/django-pldp/blob/master/pldp/models/study.py#L35). So, surveys belong to studies, and studies belong to projects, right? It seems like PLDP should have a project model (rather than just a CharField), if the project designation proves to be meaningful.

Reply:
Started a doc, and also linked it in #47 ! https://docs.google.com/document/d/1zpIHpqAn6Y9mEx4BqfDHxPEktKWHBxHyuKSt8xbFT0k/edit#

@reginafcompton feel free to add detail here if I didn't capture your thinking.